lib/prod-server.js

```diff
@@ -1,5 +1,7 @@
 const { createServer } = require('http')
+const urllib = require('url')
```

Comment:
Ah, not quite. I mean, I may have implemented it wrong, by my intention was for basePath and assetPath (Or `assetURL` then) to be separate.

So for the assetURL of `http://mycdnurl.com/baz`, and a basePath of `/foo/`, the connect server will serve static files from '/baz' and the basePath of `/foo/` will only apply to the application routing